buildSrc/src/test/java/org/springframework/boot/build/mavenplugin/PluginXmlParserTests.java

```diff
@@ -16,25 +16,49 @@
 
 package org.springframework.boot.build.mavenplugin;
 
-import java.io.File;
-
 import org.junit.jupiter.api.Test;
-
 import org.springframework.boot.build.mavenplugin.PluginXmlParser.Plugin;
 
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
 /**
  * Tests for {@link PluginXmlParser}.
  *
  * @author Andy Wilkinson
+ * @author Mike Smithson
  */
 public class PluginXmlParserTests {
 
 	private final PluginXmlParser parser = new PluginXmlParser();
 
 	@Test
-	void dunno() {
+	void parsing_a_plugin_xml_file_returns_the_gav_for_plugin() {
```

Comment:
We don't format method name that way. Please look at the rest of the codebase for inspiration.

Reply:
changed to camel case naming like other tests as of commit 976fc62